db/data_migration/20150219120320_add_historic_governments.rb

```diff
@@ -1,5 +1,5 @@
 governments = [
-  ["2010 to 2015 Conservative and Liberal democrat coalition government", "2012-05-12"],
+  ["2010 to 2015 Conservative and Liberal democrat coalition government", "2010-05-12"]
```

Comment:
Should Democrat be capitalised? Their website seems to say so http://www.libdems.org.uk/